### PR TITLE
[INFRA] Issue forms

### PR DESCRIPTION
**Pull request by Remi-Gau**
## unrelated

- update contact email
- fix typo

### CODE REVIEW

**Review by sappelhoff — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bids_apps_submission.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      options:
+        - label: complies to the BIDS app API
+        - label: has a system-test workflow set up on circle-CI
+
+# TODO
+# - add extra fields /requirements for the BIDS app:
+#   - uses raw or derivatives as inputs (or both)
+#   - outputs bids friendly derivatives (filenames follow the entity-label format)
+#   - system test on an openneuro dataset
+#   - has a boutique config
```
Comment (Remi-Gau) [inline .github/ISSUE_TEMPLATE/bids_apps_submission.yml:47]: I think that these info should be provided for every app but that will require updating the website I think.

Also I think we should allow for bids apps to run their CI with other providers, no?

Comment (sappelhoff) [reply .github/ISSUE_TEMPLATE/bids_apps_submission.yml:47]: > Also I think we should allow for bids apps to run their CI with other providers, no?

as opposed to always displaying the circleci badge? Yes, agreed.

Comment (Remi-Gau) [reply .github/ISSUE_TEMPLATE/bids_apps_submission.yml:47]: will need to tweak how the html is generated for the app page but it should not be too complicated... #FLW

Comment (sappelhoff) [reply .github/ISSUE_TEMPLATE/bids_apps_submission.yml:47]: I am resolving this now -- we discussed a bit of this here as well: https://github.com/bids-apps/bids-apps.github.io/issues/66#issuecomment-1161790955 (for x-refing)

**Comment by sappelhoff** [comment]
thanks!

**Pull request merged**